test/Microsoft.ML.StaticPipelineTesting/Microsoft.ML.StaticPipelineTesting.csproj

```diff
@@ -7,13 +7,14 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
-    <NativeAssemblyReference Include="FactorizationMachineNative" />
-
+    
     <ProjectReference Include="..\..\src\Microsoft.ML.Analyzer\Microsoft.ML.Analyzer.csproj">
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <OutputItemType>Analyzer</OutputItemType>
     </ProjectReference>
 
     <NativeAssemblyReference Include="CpuMathNative" />
+    <NativeAssemblyReference Include="FactorizationMachineNative" />
```

Comment:
I can see tests for FastTree added, but nothing regarding FM

Reply:
it was already there. I just moved the reference a few lines down, to keep it together with the other native references. 

---
In reply to: [220283652](https://github.com/dotnet/machinelearning/pull/1009#discussion_r220283652) [](ancestors = 220283652)